micro-rdk/src/common/conn/viam.rs

```diff
@@ -909,10 +986,21 @@ where
                     let (answer, prio) = api.answer(0).await?;
                     let robot = self.robot.clone();
 
-                    let task = self.executor.spawn(async move {
-                        let mut conn = api.connect(answer, robot).await?;
-                        conn.run().await
-                    });
+                    log::info!("spawning task for new WebRTC connection");
+                    let task = self.executor.spawn(
+                        async move {
```

Comment:
As above for HTTP2 but now for webrtc, only interesting changes are logging and `inspect` calls.